src/Components/Nav/Nav.jsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "./Nav.css";
+
+function Nav() {
```

Comment:
Naming for this component can be better eg Navbar or Navigation.

Reply:
Ok will do it